src/Polly/Bulkhead/Settings/BulkheadRejectedCallbackDecoratorBase.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Polly.Bulkhead.Settings;
+
+/// <summary>
+/// The <see cref="BulkheadRejectedCallbackDecoratorBase"/> is a base type which should be the base of all decorator.
+/// </summary>
+public abstract class BulkheadRejectedCallbackDecoratorBase : IBulkheadRejectedCallback
```

Comment:
I see the point, because then you are able to add a forwarding to the new method with more arguments without breaking anything - that's clever. This also shouldn't break the possibility to create a decorator.